CMakeLists.txt

```diff
@@ -26,7 +26,7 @@
 #  POSSIBILITY OF SUCH DAMAGE.
 #
 
-cmake_minimum_required(VERSION 3.13.1)
+cmake_minimum_required(VERSION 3.10.1)
```

Comment:
@wgtdkp , should we add a GitHub Action test using CMake 3.10.1?

Reply:
Yes, will do.